lib/polling.js

```diff
@@ -48,46 +52,62 @@ TradeOfferManager.prototype.doPoll = function(doFullUpdate) {
 	}
 
 	this.emit('debug', 'Doing trade offer poll since ' + offersSince + (fullUpdate ? ' (full update)' : ''));
-	this.getOffers(fullUpdate ? EOfferFilter.All : EOfferFilter.ActiveOnly, new Date(offersSince * 1000), (err, sent, received) => {
-		if (err) {
-			this.emit('debug', "Error getting trade offers for poll: " + err.message);
-			this.emit('pollFailure', err);
+
+	var filter = fullUpdate ? EOfferFilter.All : EOfferFilter.ActiveOnly;
```

Comment:
@DoctorMcKay I agree with @Aareksio on that one. It should make the code more readable. I guess allocating one small variable isn't that bad (and it would be destroyed by GC after that), but it would make the code more clear, and yes, it'd make the line shorter.